Add ports in services table (#196)
logicalclocks/hops-docs: docs/getting_started/installation_guide/platforms/custom.rst

* Prometheus    
* Jupyter

> Hopsworks uses different user accounts and groups to run services. The actual user accounts and groups needed depends on the services you install. Do not delete these accounts or groups and do not modify their permissions and rights. Ensure that no existing systems prevent these accounts and groups from functioning. For example, if you have scripts that delete user accounts not in a whitelist, add these accounts to the list of permitted accounts. Hopsworks creates and uses the following accounts and groups:

Reviewer: ```suggestion
Hopsworks uses different user accounts and groups to run services. The actual user accounts and groups needed depends on the services you install. Do not delete these accounts or groups and do not modify their permissions and rights. Ensure that no existing systems prevent these accounts and groups from functioning. For example, if you have configuration management tools, whitelist Hopsworks users/groups. Hopsworks creates and uses the following accounts and groups:
```

Author: fixed

> The table below also provides the port is service is listening at. Not all services need to accessible from

Reviewer: ```suggestion
The table below also provides the port a service is listening at. Not all services need to be accessible from
```

Author: fixed

> outside the cluster, therefore is it fine if the ports of such services are blocked by perimeter security. Services

Reviewer: ```suggestion
outside the cluster, therefore it is fine if the ports of such services are blocked by perimeter security. Services
```

Author: fixed

that need to be accessible from outside the cluster are designated in the table below.

+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| Service           |Unix User ID| Group     | Ports                | External access                                                |
+===================+============+===========+======================+================================================================+
| namenode          | hdfs       | hadoop    | 8020, 50470          | Yes, if external access to HopsFS is needed.                   |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| datanode          | hdfs       | hadoop    | 50010, 50020         | Yes, if external access to HopsFS is needed.                   |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+

> | resourcemgr       | yarn       | hadoop    | 8032                 | No                                                             |

Reviewer: RM user is `rmyarn`

Author: fixed

+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| nodemanager       | yarn       | hadoop    | 8042                 | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+

> | hopsworks         | glassfish  | glassfish | 443/8181), 4848      | Yes, only for 443/8181.                                        |

Reviewer: ```suggestion
| hopsworks         | glassfish  | glassfish | 443/8181, 4848      | Yes, only for 443/8181.                                        |
```

Author: fixed

| logstash          | elastic    | elastic   | 5044-5052, 9600      | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| filebeat          | elastic    | elastic   |        N/A           | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| kibana            | kibana     | elastic   | 5601                 | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| ndbmtd            | mysql      | mysql     | 10000                | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+

> | mysqld            | mysql      | mysql     | 3306                 | No                                                             |

Reviewer 2: Yes if external online feature store access is required.

+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| ndb_mgmd          | mysql      | mysql     | 1186                 | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+

> | hiveserver2       | hive       | hive      | 9084-9085            | No                                                             |

Reviewer 2: we only use 9085 these days. and Yes you need to open the port to the externally if you want to use SageMaker or Python integration.

Author: Is that due to PyHive or something else?

Reviewer 2: Yes it's PyHIve

+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| metastore         | hive       | hive      | 9083                 | Yes if external access to the Feature Store is needed          |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| kafka             | kafka      | kafka     | 9091, 9092(external) | Yes for 9092 if external access to the Kafka cluster is needed |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| zookeeper         | zookeeper  | zookeeper | 2181                 | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+

> | epipe             | epipe      | epipe     | 29191 (metrics)      | No                                                             |

Reviewer 2: I think we can remove this one. Otherwise we have to put the metrics port for all the services.

Author: fixed

+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| airflow-scheduler | airflow    | airflow   |        N/A           | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+
| sqoop             | airflow    | airflow   | 16000                | No                                                             |
+-------------------+------------+-----------+----------------------+----------------------------------------------------------------+

> | airflow-webserver | airflow    | airflow   | 12358                | No                                                             |

Reviewer 2: Prometheus and Grafana are notable absents to the list. Both of them are interesting if you want to plug your cluster in another monitoring service.

Reviewer 2: Sparkhistoryserver is also missing. K8s is also missing.

Author: fixed
